Backend/src/Trackable.EntityFramework/Migrations/201802261213391_AddWebhooksToGeofences.cs

```diff
@@ -0,0 +1,20 @@
+namespace Trackable.EntityFramework.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class AddWebhooksToGeofences : DbMigration
+    {
+        public override void Up()
+        {
+            AddColumn("dbo.GeoFences", "Webhooks", c => c.String());
+            AlterColumn("dbo.GeoFences", "Emails", c => c.String());
+        }
+        
+        public override void Down()
+        {
+            AlterColumn("dbo.GeoFences", "Emails", c => c.String(nullable: false));
```

Comment:
This will delete all emails on a deployment. You probably want to copy all data to an intermediary column then back.